Contribute/Markdown-Conventions/index.md

```diff
@@ -1,56 +1,62 @@
 # Markdown conventions
-The Umbraco Documentation uses Markdown for all of the documentation - but more precisely we use the CommonMark specification, read more about the difference [here](https://commonmark.org/).
+
+The Umbraco Documentation uses Markdown for all of the documentation - but more precisely we use the CommonMark specification. Read more about the [difference between CommonMark and Markdown](https://commonmark.org/).
 
 ## Structure
+
 For the documentation project, each individual topic is contained in its own folder.
 Each folder must have an `index.md` file which links to the individual sub-pages, if images are used, these must be in `images` folders next to the .md file referencing them relatively.
 
 * topic
-	* images
-		* images.jpg
-	* Subtopic
-		* images
-		* index.md
-	* index.md
-	* otherpage.md
+  * images
+    * images.jpg
+  * Subtopic
+    * images
+    * index.md
+  * index.md
+  * other-page.md
 
 ## Images
+
 Images are stored and linked relatively to .md pages, and should by convention always be in an `images` folder. So to add an image to `/documentation/reference/partials/renderviewpage.md` you link it like so:
 
-	![My Image Alt Text](images/img.jpg)
+    ![My Image Alt Text](images/img.jpg)
 
 And store the image as `/documentation/reference/partials/images/img.jpg`
 
 Images can have a maximum width of **800px**. Please always try to use the most efficient compression, `gif`, `png` or `jpg`. No `bmp`, `tiff` or `swf` (Flash).
 
 ## External links
-Include either the complete URL, or link using Markdown:
 
-	https://yahoo.com/something
+Include either the complete URL, or link using a specific syntax:
 
-	or
+    https://yahoo.com/something
 
-	[yahoo something](https://yahoo.com/something)
+or
 
+    [yahoo something](https://yahoo.com/something)
 
 ## Internal links
+
 If you need to link between pages, always link relatively, and include the .md extension.
 
-	[Umbraco.Helpers](Umbraco.Helpers.md)
+    [Umbraco.Helpers](Umbraco.Helpers.md)
 
-	or
+or
 
-	[Umbraco.Helpers](../../Reference/Umbraco.Helpers.md)
+    [Umbraco.Helpers](../../Reference/Umbraco.Helpers.md)
 
 ## Formatting code
+
 Indent your sample with 4 spaces, which will cause it to be rendered as `<pre><code>` tags.
 For inline code, wrap in ` (backtick) chars.
 
 Use # for the headline, ## for sub headers and ### for parameters (on code reference pages)
 
 For optional parameters wrap in _ (underscore) - end result: `###_optionalParameter_`
 
-## Custom Markdown
+## Adding notes, warnings, tips
```

Comment:
Adding notes, warnings, tips, checklists, whatever else will be added in the future?

Is this not too long and not scalable a title? 😄 I agree that 'Custom Markdown' is a pretty bad one, but we should figure out some name that can cover all of these things.

Reply:
Fair point, let's leave it as it is now and then revisit if / when we add more styles 😄 I was thinking it would be nice in the future to have something like a specific style for "related links" but haven't thought up how it should look yet.